diff -up UASM-2.56.2/macho64.c.omv~ UASM-2.56.2/macho64.c
--- UASM-2.56.2/macho64.c.omv~	2023-02-27 19:51:48.019071101 +0100
+++ UASM-2.56.2/macho64.c	2023-02-27 19:52:09.729173009 +0100
@@ -725,7 +725,7 @@ static ret_code macho_write_module( stru
 		mm.header.cputype = CPU_TYPE_X86_64;
 		mm.header.cpusubtype = CPU_SUBTYPE_LITTLE_ENDIAN | CPU_SUBTYPE_X86_64_ALL;
 		mm.header.filetype = MH_OBJECT;
-		mm.header.flags = NULL;
+		mm.header.flags = 0L;
 		
 		macho_build_structures(modinfo, mm);	
 	}
@@ -744,4 +744,4 @@ void macho_init( struct module_info *mod
 	modinfo->osx_osabi = MACHOABI_OSX64;
 	modinfo->g.WriteModule = macho_write_module;
 	return;
-}
\ No newline at end of file
+}
